index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Portfolio - ZC</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="icon" href="./icons/favicon.ico" type="image/x-icon">
+</head>
+<body>
+    <!-- Toolbar -->
+    <div class="toolbar">
+        <p id="toolbartext">Welcome</p>
+        <img id="menuicon" src="./icons/hamburguermenu.png" atl="Menu" />
+    </div>
+
+    <!-- Headline Section-->
+    <div class="headline">
+
+        <div class="container">
+            <h1>Hey there. I'm Cristian<br><span style="color: #36B37E"> I'm a software developer</span> </h1>
+        </div>
```

Comment:
- [ ] Kindly 😃 try to stick to the Figma Design at the color, so please try to use the correct color for the heading here to be like the Figma Design 💯

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/219170165-d4302930-a151-44c9-9ec7-0eff9cc3f26e.png)  |    ![image](https://user-images.githubusercontent.com/83360636/219169800-76eba694-3bbe-45a3-bc6f-0607b743fdb6.png)  |